src/lib/isVAT.js

```diff
@@ -3,6 +3,7 @@ import assertString from './util/assertString';
 export const vatMatchers = {
   GB: /^GB((\d{3} \d{4} ([0-8][0-9]|9[0-6]))|(\d{9} \d{3})|(((GD[0-4])|(HA[5-9]))[0-9]{2}))$/,
   IT: /^(IT)?[0-9]{11}$/,
+  DE: /^(DE)?[0-9]{9}$/

```

Comment:
As @tux-tn  has just on running lint or add a `comma here`. 
Otherwise, it looks good,